libpod/pod.go

```diff
@@ -22,7 +22,12 @@ import (
 // assume their callers handled this requirement. Generally speaking, if a
 // function takes the pod lock and accesses any part of state, it should
 // updatePod() immediately after locking.
-// ffjson: skip
+var (
```

Comment:
Why is this bisecting the comment for the Pod struct? Move it above the comment, and restore the //ffjson:skip command